platform/security/sts/security-sts-propertyclaimshandler/src/main/java/org/codice/ddf/security/sts/claims/property/PropertyFileClaimsHandler.java

```diff
@@ -48,6 +54,8 @@
 
   private String idClaimType;
 
+  private PropertiesBackingEngine pbe;
```

Comment:
✏️ I suggest using a more meaningful variable to make it more clear what we're referring to later on in the code

Reply:
Thanks, updated name